lib/Doctrine/Expression/Mysql.php

```diff
@@ -85,7 +85,7 @@ public function matchPattern($pattern, $operator = null, $field = null)
                     $match = $field.'LIKE BINARY ';
                     break;
                 default:
-                    throw new Doctrine_Expression_Mysql_Exception('not a supported operator type:'. $operator);
```

Comment:
This class doesn't exist, and this code isn't touched by tests (marked as "experimental"), just changing to an exception class that does exist.